func_brainpy_model.py

```diff
@@ -177,19 +134,18 @@ def __init__(
         I: Union[float, ArrayType] = 0.0, 
         H_x_act = 'AbbottChance', 
         S_init: Union[float, ArrayType] = None,
-        H_init: Union[float, ArrayType] = None,
-        train: list = ['G', 'w', 'I']
+        train: list = ['G', 'w', 'I'],
```

Comment:
Seems same for the original style "G_trainable: bool = True".
 I prefer the original style which is more straight forward and there are just a few parameters.